src/main/scala/it/pagopa/interop/backendforfrontend/error/Handlers.scala

```diff
@@ -54,6 +48,7 @@ object Handlers {
     case Failure(err: UnknownTenantOrigin)                      => badRequest(err, logMessage)
     case Failure(err: InvalidInterfaceFileDetected)             => badRequest(err, logMessage)
     case Failure(err: AgreementDescriptorNotFound)              => notFound(err, logMessage)
+    case Failure(err: NoDescriptorInEservice)                   => notFound(err, logMessage)
```

Comment:
In this case we can remove the handling of this exception.
This is something that should never occur, it is ok if the servire returns a 500 (so we can be notified)